app/ht_indexer/tests/ht_queue_service_tests/queue_consumer_test.py

```diff
@@ -21,141 +23,226 @@ def one_message():
 @pytest.fixture
 def list_messages():
     """
-    This function is used to create a message
+    This function is used to create a list of messages
     """
 
     messages = []
     for i in range(10):
         messages.append({"ht_id": f"{i}", "ht_title": f"Hello World {i}", "ht_author": f"John Doe {i}"})
     return messages
 
+class TestQueueConsumer:
 
-@pytest.fixture
-def populate_queue(list_messages, producer_instance, consumer_instance, retriever_parameters):
-    """ Test for re-queueing a message from the queue, an error is raised, and the message is routed
-            to the dead letter queue and discarded from the main queue"""
+    def test_queue_consume_message(self, one_message, get_rabbit_mq_host_name):
+        """ Test for consuming a message from the queue
+        One message is published and consumed, then at the end of the test the queue is empty
+        """
+
+        producer_instance = QueueProducer(
+            user= "guest", password="guest", host=get_rabbit_mq_host_name,
+            queue_name="test_queue_consume_message", batch_size=1
+        )
 
-    # Clean up the queue
-    consumer_instance.conn.ht_channel.queue_purge(consumer_instance.queue_name)
+        consumer_instance = QueueConsumer(
+            "guest",
+            "guest",
+            get_rabbit_mq_host_name,
+            "test_queue_consume_message",
+            requeue_message=False,
+            batch_size=1
+        )
+
+        # Clean up the queue
+        consumer_instance.ht_channel.queue_purge(consumer_instance.queue_name)
 
-    for message in list_messages:
         # Publish the message
-        producer_instance.publish_messages(message)
+        producer_instance.publish_messages(one_message)
 
-    start_time = time.time()
-    for method_frame, _properties, body in consumer_instance.consume_message(inactivity_timeout=3):
+        for _method_frame, _properties, body in consumer_instance.consume_message(inactivity_timeout=5):
```

Comment:
I thought prepending an underscore for e.g. `_method_frame` indicated that a variable wasn't used -- but it is used here just below. Should we remove the underscore? Or does it signify something else?